docs/products/postgresql/howto/manage-extensions.md

```diff
@@ -5,19 +5,24 @@ sidebar_label: Manage extensions
 
 Aiven for PostgreSQL® allows a series of pre-approved extensions to be installed.
 
+:::note[Who can manage extensions]
+All Aiven service users can manage extensions: both the default `avnadmin` user and other
+[service users created](/docs/platform/howto/create_new_service_user) via the Aiven
+Console, API, CLI, or Aiven Provider for Terraform. This is the expected behavior because
```

Comment:
I rather say any user in the database  instead of service user to be honest. As service users are essentially just pg roles